recipe/make_wrapper.py

```diff
@@ -0,0 +1,31 @@
+import platform
+import os
+from os.path import join
+
+
+PKG_VERSION = os.environ["PKG_VERSION"]
+
+SHELL_CONTENT = """#!/bin/bash
+java -Xmx500M -cp "$CONDA_PREFIX/lib/antlr4-%s-complete.jar:$CLASSPATH" org.antlr.v4.Tool "$@"
+""" % PKG_VERSION
+
+BAT_CONTENT = """
+java -Xmx500M -cp %CONDA_PREFIX%\\Library\\lib\\antlr4-{}-complete.jar;%CLASSPATH% org.antlr.v4.Tool %*
+IF %ERRORLEVEL% NEQ 0 EXIT /B %ERRORLEVEL%
```

Comment:
This can be done in the cmd.
```
echo java -Xmx500M -cp %LIBRARY_LIB%\\antlr4-%PKG_VERSION%-complete.jar;%%CLASSPATH%% org.antlr.v4.Tool %* > %LIBRARY_BIN%\antlr4.cmd
echo IF %%ERRORLEVEL%% NEQ 0 EXIT /B %%ERRORLEVEL%% >> %LIBRARY_BIN%\antlr4.cmd
```

Reply:
Yeah, this was the kicker that caused both of them, in that unless we hardcode the version, `PKG_VERSION` won't be available... I had trouble escaping one but not the other with various redirection incantations... can try some more...

Reply:
Yeah, the double `%%` did the trick! Thanks!